### PR TITLE
DevOps Deploy examples signer to canisters [sc-10300]

### PR DESCRIPTION
**Pull request by dmitriiIdentityLabs**

### CODE REVIEW

File: .github/workflows/deploy-example.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        run: npm install
+
+      - name: Format Check
+        run: npm run format:check
+
+      - name: Build
+        run: npm run build
+
+      - name: Test
+        run: npm run test
```
Comment (plitzenberger) [inline .github/workflows/deploy-example.yml:41]: From my point of view, we could remove them bc they had been running on PR already. And `dfx deploy` will run `npm run build` automatically, as far as I know.

File: dfx.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "build": {
+      "packtool": ""
+    }
+  },
+  "networks": {
+    "local": {
+      "bind": "127.0.0.1:8000",
+      "type": "ephemeral"
+    },
+    "ic": {
```
Comment (plitzenberger) [inline dfx.json:32]: here `prod` in the case I've mentioned above

Comment (dmitriiIdentityLabs) [reply dfx.json:32]: The akward issue is here, when I try to use prod or any other name for mainnet. The separate ticket will be created to handle this.

**Review by plitzenberger — APPROVED** [review]
awesome 🤩

**Pull request merged**